### ISSUE TITLE
Which version of Luma will this work with?

### ISSUE THREAD

**Issue by ghostmanslow** [Roberto-Nessy/OoT3D_Standalone_Free_Cam#15]
USA version of the game, tried multiple revisions. Folder is named exactly as it should be.

So far I've tried the two most recent versions of Luma and it throws an arm11 crash on booting the game. I don't really want to test version after version hoping to get the right one.

**Comment by Roberto-Nessy** [comment]
I created it with v10.2.1 installed, but just updated to v13.1.2 (latest) and it still works fine.
Since it's crashing that means the folder structure is definitely correct since it must be trying to load the patches, in which case the most likely culprits are using the wrong patch files or having some other cheats/patches active that are interfering with each other.
If that fails then it'd be helpful to see the crash screen or a dump file.

**Comment by ghostmanslow** [comment]
I downloaded the 3DS patches from the USA folder to use with the US version of the game, they're also the only patches I'm using with the game. I really appreciate your assistance with looking into this; I will attach a dump file now.
[crash_dump_00000000.dmp](https://github.com/user-attachments/files/17087832/crash_dump_00000000.dmp)

Just in case I will download the patch files again, perhaps I accidentally pulled from the wrong folder and didn't realize it.

**Comment by Roberto-Nessy** [comment]
It looks like it's failing on reading exheader.bin, I managed to produce a similar crash by trying to use the citra exheader (not completely identical though), but it worked fine again after replacing it with the 3ds one

**Comment by ghostmanslow** [comment]
Yeah, I'm definitely using the one from the 3DS folder so I'm not sure what the issue is.

You sure the right one is in there?

**Comment by ghostmanslow** [comment]
May I please have the exheader.bin file from your SD card to see if that works for me? I only ask because based on what you're saying, the one uploaded here might have some sort of issue that wasn't caught before sharing. And if it still doesn't work for me, it's some other obscure issue I'll hunt down and fix in my off time.

Would love to know for sure, thank you.

**Comment by Roberto-Nessy** [comment]
Sorry didn't see this yesterday, when I switched back I used the one from here just to be sure, but just in case these are the files I'm using.
[0004000000033500.zip](https://github.com/user-attachments/files/17120208/0004000000033500.zip)

If that doesn't work it may be worth asking in the homebrew discord as I'm not sure I can provide any more than this with my current knowledge

**Comment by ghostmanslow** [comment]
No worries, and I am very happy to report that the one you just sent works! Thank you so much!

Very weird issue though, I definitely was using the ones from the USA 3DS folder on here. Either way, I am pumped as hell for this mod so thanks again.

**Comment by lucasedng** [comment]
For people like me (who are using Luma version 13.x) and have already tried the file below without success, follow these steps:  

- Download version 10.2.1 of Luma from the official repository;  
- Install the latest version of this mod (I only needed to replace the **boot.3dsx** and **boot.firm** files);  
- Turn on the console and open the game—it should work;  
- Update to the latest version of Luma;  
- Open the game—it should be working normally.  

At least for me, this worked. I did this on my **New 3DS XL (US)** and **New 3DS (EU)**.

**Comment by MajinChibi** [comment]
@lucasedng 
Thank you for your solution.

Unfortunately, its not working for me.

New 3DS XL (EU). When i use the files from Github (EU Version) i get an error immediately.
When i use the files from this post https://github.com/Roberto-Nessy/OoT3D_Standalone_Free_Cam/issues/15#issuecomment-2372269530 (i know, its the wrong region) i get the error a little later.

What i noticed: the filesizes from https://github.com/Roberto-Nessy/OoT3D_Standalone_Free_Cam/issues/15#issuecomment-2372269530 are totally different from the filesizes on github.

Maybe you could check the patch filesizes from your EU Version? That would be awesome.
Thank you in advance.

**Comment by banana3ds** [comment]
@MajinChibi thank you I finally fixed my issue
